tcms/core/views.py

```diff
@@ -66,6 +72,31 @@ def server_error(request):  # pylint: disable=missing-permission-required
     return http.HttpResponseServerError(template.render({}, request))
 
 
+class SetupView(TemplateView):
+
+    template_name = "setup.html"
+
+    def get_context_data(self, **kwargs):
+        # Check for non-applied migrations
+        executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
+        plan = executor.migration_plan(executor.loader.graph.leaf_nodes())
+        for btn in self.request.GET:
+            if btn == "perform_migrations" and self.request.GET[btn] == "yes" and plan:
+                call_command("migrate")
+        # Check for existing superuser - not implemented yet
+        # Check if domain name is set - not implemented yet
```

Comment:
These comments aren't necessary. There's a better way to deal with the rest once you have a functional database.